### PR TITLE
dependabot: additional directories

### LINKED ISSUE #25: ci: add a dependency management bot
**Description**

Add a dependency management bot for frontend and backend packages.

One option could be to use [Dependabot](https://dependabot.com/) as it is owned by GitHub and might have better support/easier integration. Another popular tool is [Renovate](https://github.com/renovatebot/renovate).

Complexity [S/M/L]: S

### PR DESCRIPTION
**Pull request by jarz**
Also organize dependabot.yml entries by directory

### GitHub Issue

Fixes #25

### CODE REVIEW

**Comment by danielhochman** [comment]
@jarz do you by any chance have experience with renovatebot? looks like we can get it for free as a public repo and it supports globs for package discovery. i worry about keeping this up-to-date as we add more packages.

File: .github/dependabot.yml [hunk 1]
```diff
@@ -24,3 +24,93 @@ updates:
...
+
+  - package-ecosystem: "npm"
+    directory: "/tools/scaffolding/templates/frontend/"
+    schedule:
+      interval: "daily"
+
+  - package-ecosystem: "gomod"
+    directory: "/tools/scaffolding/templates/gateway/backend/"
+    schedule:
+      interval: "daily"
```
Comment (danielhochman) [inline .github/dependabot.yml:106]: the go.mod is not technically valid (syntax errors) due to the templating, i wonder how dependabot will handle that. probably same for other scaffolding package files.

Comment (jarz) [reply .github/dependabot.yml:106]: Yeah, I want to see how forgiving dependabot's parser can be.

**Comment by jarz** [comment]
@danielhochman I haven't worked with renovatebot, but I hadn't worked with dependabot until a few days ago...

I'm assuming it won't be long until the dependabot.yml file is forgotten when a new package is added, so I'm definitely open to an alternative.

I'm going to create a new issue and try a shoot-out between Renovate and Snyk (the other alternative I see frequently). How's that sound?

**Comment by danielhochman** [comment]
That sounds great.

Also, if you want, reach out to me on Slack or Twitter and I will mail you a few stickers when we get our first batch.

**Pull request closed without merging**